tests/ro-crate-metadata.json

```diff
@@ -0,0 +1,457 @@
+                {
+                    "@id": "outputs/ERR034597_2.small_fastqc.html"
+                },
+                {
+                    "@id": "stdout.log"
+                },
+                {
+                    "@id": "stderr.log"
+                }
+            ],
+            "mainEntity": {
+                "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimming_and_qc.cwl"
+            }
+        },
+        {
+            "@id": "ro-crate-metadata.json",
+            "@type": "CreativeWork",
+            "about": {
+                "@id": "./"
+            },
+            "conformsTo": [
+                {
+                    "@id": "https://w3id.org/ro/crate/1.1"
+                },
+                {
+                    "@id": "https://w3id.org/workflowhub/workflow-ro-crate/1.0"
+                }
+            ]
+        },
+        {
+            "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimming_and_qc.cwl",
+            "@type": [
+                "File",
+                "SoftwareSourceCode",
+                "ComputationalWorkflow"
+            ],
+            "attachment": [
+                {
+                    "@id": "exe/fastqc.cwl"
+                },
+                {
+                    "@id": "exe/trimmomatic_pe.cwl"
+                }
+            ],
+            "description": {
+                "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/README.md"
+            },
+            "input": [
+                {
+                    "@id": "#fastq_1"
+                },
+                {
+                    "@id": "#fastq_2"
+                }
+            ],
+            "name": "CWL_trimming_and_qc",
+            "output": [
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.1P.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.1U.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.2P.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.2U.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small_fastqc.html"
+                },
+                {
+                    "@id": "#ERR034597_2.small_fastqc.html"
+                },
+                {
+                    "@id": "stdout.log"
+                },
+                {
+                    "@id": "stderr.log"
+                }
+            ],
+            "programmingLanguage": {
+                "@id": "https://w3id.org/workflowhub/workflow-ro-crate#cwl"
+            },
+            "version": "1.0.0",
+            "yevisId": "c13b6e27-a4ee-426f-8bdb-8cf5c4310bad"
+        },
+        {
+            "@id": "https://w3id.org/cwl/vv1.0/",
```

Comment:
Hmm. We're using `ro-crate-py` to generate this object by passing metadata extracted from the given CWL file. CWL version should be described with `v` as `v1.2` as described in the documentation here:
https://www.commonwl.org/v1.2/CommandLineTool.html#CWLVersion

But in `computerlanguage.py ` it adds an extra `v` here:
https://github.com/ResearchObject/ro-crate-py/blob/c925f7c169c833619cc99993c4a8c2b20a236e19/rocrate/model/computerlanguage.py#L80

I think we should remove the `v` from the ro-crate-py function. wdyt?

Reply:
Adding a leading "v" is common usage, but it's not an actual part of the version string. See https://peps.python.org/pep-0440/#preceding-v-character. Anyway, I've [changed ro-crate-py](https://github.com/ResearchObject/ro-crate-py/pull/154) to generate the correct URL both if the version has or hasn't the leading "v". This will be included in a future release. In the meantime, please provide the version string without the leading "v".